### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1] Radio button: State and association not provided

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11274]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/platform/table/basic
Table with Sort and Group:
The radio buttons within the dialog 'sort' are not associated to its group header 'Sort order/Sort by' and UI type as radio button not mention and also the status 'Selected/not selected' is also not conveyed.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using Screen reader can check by implementing similar case. Just use table with sort and group on toolbar. click sort/group icon .to open there respective dialog popup. The radio buttons within the dialog 'sort' are not associated to its group header 'Sort order/Sort by' and UI type as radio button not mention and also the status 'Selected/not selected' is also not conveyed.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

**Comment by mikerodonnell89** [comment]
Fixed as part of #12525
